resolwe_bio/processes/reads_processing/markduplicates.py

```diff
@@ -54,6 +53,7 @@ class Input:
                         'otherwise need to be decoded. Default is STRICT.',
             choices=[
                 ('STRICT', 'STRICT'),
+                ('LENIENT', 'LENIENT'),
```

Comment:
@romunov should this choice be also exposed in the workflow inputs?

Reply:
Yes, I missed that one.

Reply:
We'll do that for the next release.